sdk/spring/azure-spring-boot/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 3.0.0-beta.2 (Unreleased)
 ### Breaking Changes
 - Deprecate `AADAppRoleStatelessAuthenticationFilter` and `AADAuthenticationFilter`.
+- Support the setting of post logout redirect uri 
```

Comment:
Why is this a breaking change instead of a new feature? And by the way, do we need to add this in the CHANGELOG of the aad starter? 
@chenrujun